app/assets/javascripts/discourse/app/templates/modal/delete-topic-confirm.hbs

```diff
@@ -0,0 +1,13 @@
+{{#d-modal-body}}
+  <p>{{html-safe (i18n "post.controls.delete_topic_confirm_modal" minViews=minViews)}}</p>
```

Comment:
There's no HTML in this key so I don't think it should be passed through the `html-safe` helper.